### PR TITLE
hari test

### PR DESCRIPTION
**Pull request by haridira**

### CODE REVIEW

**Comment by haridira** [comment]
check vincent/char

File: src/_assets/scss/_block-listing.scss [hunk 1]
```diff
@@ -16,6 +16,34 @@
   border: 1px solid $colorOrange;
   padding: 10px;
   margin: 0;
+  background-color: white;
+  
+  // transition: border 300ms;
+  // transition: color 11100ms;
+  // -webkit-transition: color 11100ms;
+  // transition: transform 300ms;
```
Comment (vincentntang) [inline src/_assets/scss/_block-listing.scss:24]: should remove these comments before merging

File: src/_assets/scss/_block-listing.scss [hunk 2]
```diff
@@ -16,6 +16,34 @@
...
+      color: $colorGreen;
+    }
+  }
+ 
+  // a {
+  //   border-bottom: 3px solid $colorOrange;
+  // }
+  // a:hover {
+  //   border-bottom-color: 3px solid $colorGreen;
+  // }
```
Comment (vincentntang) [inline src/_assets/scss/_block-listing.scss:46]: same with this

File: src/_includes/layouts/home.njk [hunk 3]
```diff
@@ -22,19 +22,18 @@ layout: layouts/base.njk
     <div class="td-header__ctas">
       {# <a class="js-moveto" href="#meetups">Meetups</a> #}
       {# <a target="_blank" rel="noopener noreferrer" href="https://meetup.com/tampadevs">Meetup</a>
-            <a target="_blank" rel="noopener noreferrer" href="https://join.slack.com/t/tampa-devs/shared_invite/zt-veftezkg-kq~jFtC1FCz4o6suybMl5Q">Slack</a> #}<a class="js-moveto" href="#meetup">Meetup</a>
-      <a class="js-moveto" href="#slack">Slack</a>
+            <a target="_blank" rel="noopener noreferrer" href="https://join.slack.com/t/tampa-devs/shared_invite/zt-veftezkg-kq~jFtC1FCz4o6suybMl5Q">Slack</a> #}<a class="js-moveto" href="#meetup">Events</a>
```
Comment (vincentntang) [inline src/_includes/layouts/home.njk:25]: remove this anchor element comment

**Review by vincentntang — APPROVED** [review]

**Pull request merged**